### PR TITLE
docs: added Contributors — The Golden Metric of OpenSource Projects blog

### PR DESCRIPTION
**Pull request by Serendipity96**
Changes:

docs: added Contributors — The Golden Metric of OpenSource Projects blog

### CODE REVIEW

File: website/blog/2021-08-14-contributors-the-golden-metric-of-openSource-projects-en.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
+---
+title: "Contributors — The Golden Metric of OpenSource Projects"
+author: Lien
+authorURL: "https://medium.com/@lilien1010/about"
+authorImageURL: "https://miro.medium.com/fit/c/1360/1360/1*cNmhlRdcWxuom1cbm5ZS4g.jpeg"
```
Comment (juzhiyuan) [inline website/blog/2021-08-14-contributors-the-golden-metric-of-openSource-projects-en.md]: use avatar from https://github.com/lilien1010

**Review by juzhiyuan — APPROVED** [review]

File: website/blog/2021-08-14-contributors-the-golden-metric-of-openSource-projects.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+目前贡献者图表已经在 Apache APISIX 、Apache Skywalking、Apache DolphinScheduler、Apache Openwhisk、Apache ShardingSphere、awesome-docker、TiDB docs-dm 等多个开源项目中使用。
+
+在用户使用并反馈的过程中，我们实现了更多的需求，比如添加“匿名”贡献者，或是项目由 SVN 迁至 GitHub 后添加 SVN 侧的贡献者等。
+
+好程序都是起源于程序员要解决的切身之痛。当我们聊 Contributor Graph 时，我们不仅希望借助这个工具生产更加直观的展示 Apache APISIX 社区活跃度的图表，我们同时开源了这个小工具，希望这个工具可以帮助到其他的开源项目。
+
+欢迎大家使用贡献者趋势图表来跟踪您的社区活跃度，任何需求和问题都欢迎到 Contributor Graph 的 GitHub 仓库中反馈。
+
+您可以通过以下链接或者点击阅读原文，访问 Contributor Graph [repository](https://github.com/api7/contributor-graph)。
```
Comment (liuxiran) [inline website/blog/2021-08-14-contributors-the-golden-metric-of-openSource-projects.md]: it seems that we do not have `阅读原文` link in our blog, how about only leave the link at the end of the page

Comment (Serendipity96) [reply website/blog/2021-08-14-contributors-the-golden-metric-of-openSource-projects.md]: thanks

**Review by juzhiyuan — APPROVED** [review]

**Review by liuxiran — APPROVED** [review]
LGTM

**Pull request merged**